templates/style.php

```diff
@@ -138,17 +138,17 @@
 
 /* Titlebar */
 nav.amp-wp-title-bar {
-	background: #0a89c0;
+	background: <?php echo get_theme_mod( 'amp_navbar_background' ); ?>;
 	padding: 0 16px;
 }
 
 nav.amp-wp-title-bar div {
 	line-height: 54px;
-	color: #fff;
+	color: <?php echo get_theme_mod( 'amp_navbar_color' ); ?>;
 }
 
 nav.amp-wp-title-bar a {
-	color: #fff;
+	color: <?php echo get_theme_mod( 'amp_navbar_color' ); ?>;
```

Comment:
> We should probably pass these through template data instead (to allow for more easier filtering)
Not sure I'm following. Customizer settings are stored in the options table. Can you clarify what you're talking about?

> Also, I'm not sure if get_theme_mod already sanitizes the values, but if not, we should probably escape them on output.
The value is sanitized on save via the sanitization callback. In the case of the hex codes, that's `sanitize_hex_color()`. We could probably escape, though it's worth mentioning that even the default themes in core don't both escaping the values on output.